package/network/utils/iptables/Makefile

```diff
@@ -611,7 +626,6 @@ endef
 
 define Package/ip6tables/install
 	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/ip6tables{,-restore,-save} $(1)/usr/sbin/
```

Comment:
Please mention in the comit message that this just installs symlinks and that the ALTERNATIVES attribute does more or less the same.

Reply:
done